local/network.go

```diff
@@ -379,6 +392,9 @@ func (ln *localNetwork) addNode(nodeConfig node.Config) (node.Node, error) {
 		fmt.Sprintf("--%s=%s", config.BootstrapIDsKey, ln.bootstrapIDs),
 	}
 
+	for flagName, flagVal := range nodeConfig.Flags {
+		flags = append(flags, fmt.Sprintf("--%s=%v", flagName, flagVal))
+	}
```

Comment:
We should log a warning if the user overrides `config.NetworkNameKey`, `config.BootstrapIPsKey` or `config.BootstrapIDsKey` because not using the ones provided by the network runner is probably going to result in the node not working. We could just disallow the usage of those flags, too. What do you think @holisticode ?

Reply:
We could have an array of unsupported flags, and if any of these are provided we just exit.

Reply:
@holisticode Can you please add a warning log that's printed when the user sets one of the above flags? Other than that, LGTM :)

Reply:
Done